update_command.php

```diff
@@ -37,6 +39,9 @@
 
     // modules
     $modules = supported_modules($cmsMajor);
+    if ($cmsMajor === CURRENT_CMS_MAJOR) {
+        $modules = array_merge($modules, extra_modules());
+    }
```

Comment:
Please add a comment in code about why this is only for current major.
I was going to just ask you to explain it to me directly but a code comment seems better for this as a) I'm not the only person likely to be confused and b) I'll be confused about it next time I see it anyway lol

Reply:
Updated.  I also renamed it from extra_modules() to extra_repositories() since gha-* aren't modules